en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -69,62 +62,77 @@ isolated function getSupportedCurrencies() returns string[]|error {
 
 The resource invocations between services need to be done using [client access actions](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.2.0#support-for-resource-methods-in-client-objects).
 
-## Explore diagram features
+## Open the Project Design view
+
+> **Example project**
+>
+> Let's use the [Google Cloud Platform demo](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) to demonstrate the different capabilities of the Design Diagram. This sample has 10 services (9 GRPC and one HTTP) in 10 separate Ballerina packages. The `ui folder` is not a Ballerina package, hence would not make an impact. 
+
+1. To set up a Ballerina project as a VS Code workspace, add all the Ballerina packages to the workspace using one of the following methods:
+
+      -  Use the **VS Code workspace file** from an already created workspace to import its packages to a new workspace.
+
+         > **Info:** The workspace file of the [Google Cloud Platform demo](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) project is available in the repository.
+
+      -  Add each Ballerina package to the workspace using the **Add folder to workspace** option in the VS Code editor.
 
-The [Google Cloud Platform demo](https://github.com/ballerina-guides/gcp-microservices-demo/tree/project-design-demo) is used to demonstrate the different capabilities of the diagram. This sample has 10 services (9 GRPC and one HTTP) in 10 separate Ballerina packages. The `ui folder` is not a Ballerina package, hence would not make an impact.
+      When the workspace is set up, you will see the following folder structure in the **Explorer**:
 
-As per the [Related concepts](#a-project-workspace) section, each of these packages needs to be loaded into a VS Code workspace. This could be done using any of the following approaches.
-1. Use the VS Code workspace file that is available in the repository
-2. Add each Ballerina package to the workspace using the `Add folder to workspace` feature available in VS Code
+      >**Note:** The workspace name may be different if you set up the workspace from scratch.
 
-If the workspace has been set up accurately, you would be able to see the following folder structure in the explorer.
->**Note:** The workspace name would be different if you set up the workspace from scratch.
+      <img src="../../img/pdd-workspace-structure.png" alt="Workspace Structure" width="400"/>
```

Comment:
The image path seems incorrect.

Reply:
When we use the html syntax, an extra `../` seems to be required. Works locally.
I added the html path here to be able to resize the image.

Reply:
Doesn't load in md preview option in github file.

Reply:
yeah.. that can sometimes be an issue. But these will work in the build.
Shall we address this separately?